index.js

```diff
@@ -198,9 +198,23 @@ var publicAgent = {
   private_: function() { return agent; }
 };
 
-module.exports = global._google_trace_agent = publicAgent;
+/**
+ * @param {object} options - These are currently ignored
+ */
+function Trace(options) {
+  if (!(this instanceof Trace)) {
+    return new Trace(options);
+  }
+}
+
+Trace.prototype.startAgent = function(projectConfig) {
+  publicAgent.startAgent(projectConfig);
+  return publicAgent;
+};
+
+module.exports = global._google_trace_agent = Trace;
 
 // If the module was --require'd from the command line, start the agent.
 if (module.parent && module.parent.id === 'internal/preload') {
-  module.exports.start();
+  module.exports().startAgent();
```

Comment:
The module now exports a constructor that is used to make `Trace` objects.  Each `Trace` object then has a single method, `startAgent()`, that is used to start the agent (and the agent is returned by `startAgent()`).